src/components/dashboard/menu-table/menu-table.js

```diff
@@ -93,6 +74,76 @@ const MenuControls = styled.div`
     }
 `
 
+const EditModeButton = styled.div`
+  display: ${ props => (props.showEditMode ? "none" : "block")};
+  background: #628DEB;
+  box-shadow: 0px 10px 20px rgba(83, 131, 236, 0.2);
+  border-radius: 6px;
+  text-align: center;
+  padding: 10px 20px;
+  cursor: pointer;
+  z-index: 10;
+  width: 50px;
+`
+
+const EditModeDoneButton = styled.div`
+  display: ${ props => (props.showEditMode ? "block" : "none")};
+  background: #628DEB;
+  box-shadow: 0px 10px 20px rgba(83, 131, 236, 0.2);
+  border-radius: 6px;
+  text-align: center;
+  padding: 10px 20px;
+  cursor: pointer;
+  z-index: 10;
+  width: 50px;
+`
+
+const CopyNewMenuButton = styled.div`
+  display: ${ props => (props.showEditMode ? "block" : "none")};
+  background-color: #F3A35C;
+  padding: 10px 20px;
+  text-align: center;
+  border: 2px solid #F3A35C;
+  border-radius: 6px;
+  margin-left: 10px;
+  margin-right: 10px;
+  cursor: pointer;
+  width: 200px;
+`
+
+const NewCategoryButton = styled.div`
+  display: ${ props => (props.showEditMode ? "none" : "block")};
+  background-color: #F3A35C;
+  padding: 10px 20px;
+  text-align: center;
+  border: 2px solid #F3A35C;
+  border-radius: 6px;
+  margin-left: 10px;
+  margin-right: 10px;
+  cursor: pointer;
+  width: 200px;
+`
+
+const NewDishButton = styled.div`
+  display: ${ props => (props.showEditMode ? "none" : "block")};
+  border: 2px solid #F3A35C;
+  padding: 10px 20px;
+  color: #F3A35C;
+  border-radius: 6px;
+  cursor: pointer;
+  width: 100px;
+`
+
+const DeleteButton = styled.div`
+  display: ${ props => (props.showEditMode ? "block" : "none")};
+  border: 2px solid #FA3838;
+  padding: 10px 20px;
+  background: #FA3838;
+  border-radius: 6px;
+  cursor: pointer;
+  width: 100px;
+`
+
```

Comment:
Can we consolidate these button stylings?  Looks like these are all basically the same styling

Reply:
fixed